CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ project(KinesisVideoProducerC LANGUAGES C)
 # User Flags
 option(ADD_MUCLIBC "Add -muclibc c flag" OFF)
 option(BUILD_DEPENDENCIES "Whether or not to build depending libraries from source" ON)
-option(BUILD_OPENSSL "If building dependencies, whether or not building openssl from source" OFF)
+option(BUILD_OPENSSL "If building dependencies, whether or not building openssl from source" ON)
```

Comment:
Is this change related to what the current patch intends to do?

Reply:
removed.